### ISSUE TITLE
Error hint formatting in settings hard to parse

### ISSUE THREAD

**Issue by seanmcbreen** [Microsoft/vscode#35147]
Testing #34769 

I noticed that the hover error for the enum accepted in a setting is pretty hard to read - it's accurate for sure but could we format it better for a long list like this one?  Even something as simple as not wrapping mid values could help. 

This is a low priority issue and can be closed if its just too tricky.

![foo](https://user-images.githubusercontent.com/1051190/30874366-f0ff1916-a2a4-11e7-93c2-6177e5caed17.gif)

Sean

**Comment by aeschli** [comment]
It's a long list so stretching it (e.g having each item on it's own line) would this a very long hover, probably even more impractical.

I'll remove the '[', and file an issue against to hover to not wrap in the middle of a word.

**Comment by aeschli** [comment]
Closing.
